src/simulators.jl

```diff
@@ -860,7 +867,7 @@ function simulate!(sys::System{D, G, T},
                    sim::MetropolisMonteCarlo,
                    n_steps::Integer;
                    n_threads::Integer=Threads.nthreads(),
-                   log_states::Bool=true) where {D, G, T}
+                   run_loggers = true) where {D, G, T}
```

Comment:
Remove the reference to `log_states` from the docstring. Also there is a test that calls `log_states` which will need to change.